src/styles/blocks/card.css

```diff
@@ -0,0 +1,91 @@
+.card {
+  width: 200px;
+  height: 406px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  padding: 16px;
+}
+  
+.card__img {
+  width: 160px;
+  height: 134px;
+  object-fit: contain;
+  object-position: center;
+  display: inline-flex;
+  margin: 16px 3px 40px;
+}
+  
+.card__title {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  font-family: inherit;
+  color: #060b35;
+  font-style: normal;
+  vertical-align: top;
+  text-align: left;
+  padding-bottom: 4px;
+}
+  
+.card__type {
+  font-family: inherit;
+  font-size: 10px;
+  font-weight: 400;
```

Comment:
Remove redundant default value

Reply:
is not fixed

Reply:
still not fixed, if there are some issues you can leave a comment about reason why you do it here, or ask us in the slack if you don't understand what we are asking about

Reply:
ff inherit was in my previous commit not in the last one.